certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1085 @@
+                },
+                "EnterpriseId": {
+                  "type": "array",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "description": "EnterpriseId"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search Enterprise ID",
+        "description": "Search the Tesseron ASM enterpriseID with the customer number.",
+        "operationId": "SearchEnterprise",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "referenceNumber": {
+                  "type": "string",
+                  "description": "Please provide the business partner ID",
+                  "title": "Business partner ID"
+                }
+              },
+              "required": [
+                "referenceNumber"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/ApiEnterprise/GetEnterprise": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "EnterpriseId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The enterprise ID of the requested enterprise.",
+                  "title": "Enterprise ID"
+                },
+                "EnterpriseName": {
+                  "type": "string",
+                  "description": "The enterprise name of the requested enterprise.",
+                  "title": "Enterprise name"
+                },
+                "BusinessPartnerId": {
+                  "type": "string",
+                  "description": "The business partner ID of the requested enterprise.",
+                  "title": "Business partner ID"
+                },
+                "MailDomain": {
+                  "type": "string",
+                  "description": "The mail domain of the requested enterprise.",
+                  "title": "Mail domain"
+                },
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "Status": {
+                  "type": "string",
+                  "description": "Status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get enterprise information",
+        "operationId": "GetEnterprise",
+        "description": "Get the main enterprise information (enterpriseID, businessPartnerID & domain)",
```

Comment:
@developmentLL same here. If you want the CaMel case strings to stay as they are, then it's fine. If you want to localize them, please fix the casing.